domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Controllers/TransferAgreementProposalsControllerTests.cs

```diff
@@ -357,7 +359,11 @@ public async Task GetTransferAgreementProposal_ShouldReturnNotFound_WhenTransfer
 
         var client = factory.CreateAuthenticatedClient(sub: sub, tin: tin);
 
-        var response = await client.GetAsync($"api/transfer/transfer-agreement-proposals/{invitationId}");
+        var retryPolicy = Policy
```

Comment:
What is this? And has it something to do with sdk/runtime upgrade?

Reply:
Possibly. It's a flaky test, i just fixed. I think it has always been flaky. So I just boy scout fixed it.